### DISCUSSION TITLE
Error initializing Kamon

### DISCUSSION THREAD

**Discussion by fherrera-automox** [kamon-io/Kamon#909] [Q&A]
Hi, starting this discussion as suggested on gitter:

**Francisco H @fherrera-automox_gitlab Nov 27 12:07**
Hi, I'm new to Kamon and got stuck trying to use it first time. I downloaded the Akka-Quickstart example (https://developer.lightbend.com/guides/akka-quickstart-java/) and then added Kamon following the instructions here - https://kamon.io/get-started/ ... when trying to run both command line and on IntelliJ I am getting this error:

_Task :run FAILED
Exception in thread "main" java.lang.NoSuchMethodError: scala.collection.Seq$.empty()Lscala/collection/GenTraversable;
at kamon.Configuration.$init$(Configuration.scala:28)
at kamon.Kamon$.<init>(Kamon.scala:19)
at kamon.Kamon$.<clinit>(Kamon.scala)
at kamon.Kamon.init(Kamon.scala)
at com.example.AkkaQuickstart.main(AkkaQuickstart.java:9)
FAILURE: Build failed with an exception._

**SimunKaracic @SimunKaracic 01:00**
What version of Kamon are you using? How are you running the app?

I just downloaded the quickstart app, added the kamon-bundle dependency to `build.sbt, and I'm not getting any errors. If you could share the code, that would help too

**SimunKaracic @SimunKaracic 07:25**
You can always pop by github and open a Discussion, gitter won't be here forever! And it's way harder to search for old questions/answers https://github.com/kamon-io/Kamon/discussions

**Comment by fherrera-automox** [comment]
Hi  @SimunKaracic, the code looks like this (exactly as downloaded):

```
package com.example;

import akka.actor.typed.ActorSystem;
import kamon.Kamon;

import java.io.IOException;
public class AkkaQuickstart {
  public static void main(String[] args) {
    Kamon.init();

    //#actor-system
    final ActorSystem<GreeterMain.SayHello> greeterMain = ActorSystem.create(GreeterMain.create(), "helloakka");
    //#actor-system

    //#main-send-messages
    greeterMain.tell(new GreeterMain.SayHello("Charles"));
    //#main-send-messages

    try {
      System.out.println(">>> Press ENTER to exit <<<");
      System.in.read();
    } catch (IOException ignored) {
    } finally {
      greeterMain.terminate();
    }
  }
}
```

**Comment by SimunKaracic** [comment]
I followed the same instructions and it's running fine with maven, gradle and sbt.

What version of java/scala/kamon are you using? 
How are you running the app?
